packages/core/src/dialects/snowflake/query-generator-typescript.ts

```diff
@@ -14,6 +14,10 @@ export class SnowflakeQueryGeneratorTypeScript extends AbstractQueryGenerator {
   showConstraintsQuery(tableName: TableNameOrModel, options?: ShowConstraintsQueryOptions) {
     const table = this.extractTableDetails(tableName);
 
+    if (options?.columnName) {
```

Comment:
I think we can replace this with usage of `rejectInvalidOptions` to make it more unified with the other functions

Reply:
Done